extensions/ql-vscode/src/vscode-tests/test-helpers.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path';
```

Comment:
Can I make a suggestion that we start using more specific util modules? e.g. `file-utils.ts` rather than `test-helpers` which could be a bucket for pretty much anything.